### ISSUE TITLE
Discovery: Allow editors to easily produce accessible content from the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10274]
## Background
In order to support an accessible front-end VA.gov experience, we need to make sure our editors are able to easily produce accessible content within the CMS.

## Additional Context
This work will support ATAG 2.0 Part B: Support the production of accessible content

## Acceptance Criteria
- [ ] Research & Identify areas within the CMS that consistently result in editor-produced accessibility errors
- [ ] Determine possible additions to the CMS to identify errors before they are published/introduced to the front-end website

**Comment by laflannery** [comment]
- Can we update/improve the inline guidance on link fields?
- Can we improve the link widget within the WYSIWWYG? Will CKEditor 5 provide any additional options for this?
- Dave and I have discussion editor training to assist with the heading violations issues and in my head I also envision including link creation in this training
  - [This](https://docs.google.com/presentation/d/14AwUoFOg_Ix1Txl9oePtPD1ntOpylBWwBJMYq9f5vmo/edit#slide=id.ge89223ffe9_0_0)might actually be helpful for that training
- The KB has articles on link creation but I do not think the accessibility implications are prominent enough. We are already updating the KB so we just need to focus on the links sections when we get there.

**Comment by EWashb** [comment]
I think this would be the next step after we look through all the possibilities in SiteImprove

**Comment by laflannery** [comment]
I agree, and some small things we have done do apply to this (i.e. alt text, the discovery ticket we have to possibly improve the edit form for headings, etc.) but we can look at this more as a whole
